articles/Big Corporations and Bitcoin.txt

```diff
@@ -0,0 +1,25 @@
+|   |   |
+|---|---|
+| Title               | Could big corporations control Bitcoin? |
+| Author              | João Paulo Sant'Anna |
+| Featured text       | FIXME |
+| CT link             | FIXME |
+| CT date             | YYYY-MM-DD FIXME |
+| CT tweet            | FIXME |
+| YouTube link        | FIXME |
+| YouTube tweet       | FIXME |
+| Twitter full video  | FIXME |
+| Twitter clip 1      | FIXME |
+| Twitter clip 2      | FIXME |
+
+↑ Add more table rows for more social media links. See [this completed article](articles/simple-payment-verification.md) for a real example.
+
+Thesis statement - Will Bitcoin Succumb to Corporate Hegemony, or Will Decentralization Prevail?
```

Comment:
What arguments are you looking at here? Is this a follow on from your ETF article (https://www.cypherpunktimes.com/navigating-the-secs-bitcoin-etf-approval-through-cypherpunk-eyes/) if so, what new points are you bringing to the discussion?

Just a few thoughts:
* What is your definition of decentralisation? 
* Which corporations are aiming to control BTC and what thought do you have on how they intend to do this?

Reply:
Tanks for the comment!

1- For decentralisation I like the definition that takes decision making in consideration. Like "Lack of reliance on a single central point of command or administration."

2- For this article i would like to highlight 3 corporations: 

-BlackRock
-Fidelity
-U.S government (not a corporation but one organization that holds a lot of BTC)

Reply:
point 1 should be interesting. How do you perceive the three entities listed will have an impact on Bitcoin's decision making process?

Is there any evidence that any of these entities are already participating in the decision making or admin of BTC or is it theoretical?

If the focus is on decentralised decision making maybe the thesis statement needs to be more focus towards this. maybe something like "Putting Bitcoin's rough consensus to the test. Will it Prevail? or will it Succumb to Corporate Hegemony?"

Reply:
Thanks very much. I think thesis statement that you suggested fits just right. For now it's just theoretical, but Bitcoin and Fidelity already holds 0.5% of the total Bitcoin supply, and they are still buying it. They surely have ways to influence over the price

Reply:
Do you think i should add anything about the mining centralization also?